scripts/script.js

```diff
@@ -0,0 +1,186 @@
+const container = document.querySelector(".container");
```

Comment:
Good to see that you have not used var to create any variables and nice use of const and let keywords. Good Job